data/src/androidTest/java/org/cryptomator/data/db/UpgradeDatabaseTest.kt

```
@@ -915,6 +916,97 @@ class UpgradeDatabaseTest {
+		try {
+			Sql.insertInto("UPLOAD_CHECKPOINT_ENTITY") //
+				.integer("_id", 2) //
+				.integer("VAULT_ID", 42) //
+				.text("TYPE", "folder") //
+				.text("TARGET_FOLDER_PATH", "/other") //
+				.text("COMPLETED_FILES", "[]") //
+				.integer("TOTAL_FILE_COUNT", 2) //
+				.integer("TIMESTAMP", 2000) //
+				.executeOn(db)
+			Assert.fail("Expected constraint violation for duplicate VAULT_ID")
+		} catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)